pkg/smart/checkRequiredTools_darwin.go

```diff
@@ -0,0 +1,27 @@
+// +build darwin
```

Comment:
Please use snake-case instead camel-case for filename

Reply:
done